qml/pages/CameraUI.qml

```diff
@@ -37,296 +38,444 @@ Page {
 
     focus: true
 
-    Rectangle {
-        parent: window
-        anchors.fill: parent
-        z: -1
-        color: "black"
-    }
-
-    FSOperations {
-        id: fsOperations
-    }
+    Item {
+        id: outerRotation
+        rotation: _rotationValues["ui"][page.orientation]
+        width: page.orientation === Orientation.Portrait || page.orientation === Orientation.PortraitInverted ? parent.height : parent.width
+        height: page.orientation === Orientation.Portrait || page.orientation === Orientation.PortraitInverted ? parent.width : parent.height
+        anchors.centerIn: parent
 
-    VideoOutput {
-        id: captureView
-        anchors.fill: parent
-        source: camera
-        rotation: camera.position == Camera.FrontFace ? 0 : 180
-        orientation: camera.orientation
-        onOrientationChanged: {
-            console.log(orientation);
+        Rectangle {
+            parent: window
+            anchors.fill: parent
+            z: -1
+            color: "black"
         }
-    }
 
-    GridOverlay {
-        aspect: settings.global.captureMode == "image" ? ratio(camera.imageCapture.resolution) : ratio(camera.videoRecorder.resolution)
-
-        function ratio(resolution) {
-            return resolution.width / resolution.height
+        FSOperations {
+            id: fsOperations
         }
-    }
 
-    Slider {
-        id: zoomSlider
-        anchors.horizontalCenter: parent.horizontalCenter
-        anchors.bottom: parent.bottom
-        width: parent.width * 0.75
-        minimumValue: 1
-        maximumValue: camera.maximumDigitalZoom
-        value: camera.digitalZoom
-        stepSize: zoomStepSize
-        rotation: {
-            // Zoom slider should be slide up to zoom in
-            if (camera._orientation  === 1 ) return -180
-            else if (camera._orientation === 2) return 0
-            else return controlsRotation }
-
-        onValueChanged: {
-            if (value != camera.digitalZoom) camera.digitalZoom = value;
+        VideoOutput {
+            id: captureView
+            anchors.fill: parent
+            source: camera
+            rotation: camera.position == Camera.FrontFace ? 0 : 180
+            orientation: camera.orientation
+            onOrientationChanged: {
+                console.log(orientation);
+            }
         }
 
-        Connections {
-            target: camera
+        GridOverlay {
+            aspect: settings.global.captureMode == "image" ? ratio(camera.imageCapture.resolution) : ratio(camera.videoRecorder.resolution)
```

Comment:
I'll address the == vs === things in a future PR, no worries about those ;)

Reply:
I think that was just an indentation change